File: src/kyu8/arraySum.cs
// CSharp

public static class Kata
{
  public static int ArrayPlusArray(int[] arr1, int[] arr2)
  {
    int sol = 0;
    for(int i = 0; i < arr1.Length; i++){sol += arr1[i];}
    for(int i = 0; i < arr2.Length; i++){sol += arr2[i];}
    return sol;
  }
}

File: src/kyu8/arraySum.js
// JavaScript
function arrayPlusArray(arr1, arr2) {
  arr3 = [];
  for(var i = 0; i < arr1.length; i++){arr3.push(arr1[i] + arr2[i]);}
  return arr3.reduce((a, b) => a + b, 0);
}
